recipes/bimg/all/conanfile.py

```diff
@@ -0,0 +1,251 @@
+from conan import ConanFile
+from conan.tools.files import copy, get, rename
+from conan.tools.build import check_min_cppstd
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc, check_min_vs, is_msvc_static_runtime
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import MSBuild, VCVars
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.env import VirtualBuildEnv
+from pathlib import Path
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class bimgConan(ConanFile):
+    name = "bimg"
+    license = "BSD-2-Clause"
+    homepage = "https://github.com/bkaradzic/bimg"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Image library providing loading, saving, conversions and other utilities."
+    topics = ("image", "graphics")
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"fPIC": [True, False], "tools": [True, False]}
+    default_options = {"fPIC": True, "tools": False}
+
+    @property
+    def _bx_folder(self):
+        return "bx"
+
+    @property
+    def _bimg_folder(self):
+        return "bimg"
+
+    @property
+    def _bimg_path(self):
+        return os.path.join(self.source_folder, self._bimg_folder)
+
+    @property
+    def _genie_extra(self):
+        genie_extra = ""
+        if is_msvc(self) and not is_msvc_static_runtime(self):
+            genie_extra += " --with-dynamic-runtime"
+        if self.options.tools:
+            genie_extra += " --with-tools"
+        return genie_extra
+
+    @property
+    def _lib_target_prefix(self):
+        if self.settings.os == "Windows":
+            return "libs\\"
+        else:
+            return ""
+
+    @property
+    def _tool_target_prefix(self):
+        if self.settings.os == "Windows":
+            return "tools\\"
+        else:
+            return ""
+
+    @property
+    def _projs(self):
+        projs = [f"{self._lib_target_prefix}bimg", f"{self._lib_target_prefix}bimg_decode", f"{self._lib_target_prefix}bimg_encode"]
+        if self.options.tools:
+            projs.extend([f"{self._lib_target_prefix}texturec"])
+        return projs
+
+    @property
+    def _compiler_required(self):
+        return {
+            "gcc": "8",
+            "clang": "3.3",
+            "apple-clang": "5",
+        }
+
+    @property
+    def _bx_version(self): #mapping of bimg version to required/used bx version
+        return {"cci.20230114": "cci.20221116"}
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires(f"bx/{self._bx_version[self.version]}@")
```

Comment:
```suggestion
        self.requires(f"bx/{self._bx_version[self.version]}")
```
The `@` is not needed when adding as requirement. Only when using your terminal you need to add `@` to justify that your asking for `<name/version>` and not `<user/channel>`

Reply:
I see, good to know! I'll remove that.